secator/tasks/ffuf.py

```
@@ -93,7 +93,7 @@ def on_cmd_opts(self, opts):
 			opts['header']['value']['Host'] = f'FUZZ.{host}'
 			if self.get_opt_value('wordlist') == 'http':
 				self.add_result(Info(message='Changing wordlist to http_params as the default http wordlist is not suitable for fuzzing host header'))  # noqa: E501
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)